.github/workflows/daily-test.yml

```diff
@@ -0,0 +1,55 @@
+name: Daily Test
+
+on:
+  schedule:
+    # More information on cron https://crontab.guru/
+    # GitHub actions is using UTC time. Scheduling action at 5 am PST
+    - cron: '0 13 * * *'
+
+  workflow_dispatch:
+
+env:
+  SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+  SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  daily-test:
+    name: Amazon Chime iOS SDK Daily Test
+    runs-on: ubuntu-latest
+    outputs:
+      job-status: ${{ job.status }}
+
+    steps:
+      - name: Checkout test package
+        uses: actions/checkout@v4
+        with:
+          repository: awslabs/amazon-chime-sdk-apple
+          token: ${{ secrets.GH_INTEG_READ_ONLY_PAT }}
+          ref: automated-test-development
+
+      - name: Get latest prod demo app
+        run: |
+          wget -O amazon-chime-sdk-app.ipa ${{ secrets.DEMO_APP_DOWNLOAD_LINK }}
+      - name: Setup Node.js - 15.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 15.x
```

Comment:
node 18 or 20?

Reply:
the testing package right now only support Node 15 and less, will update once we update the testing package